packages/ListBox/src/components/NoResults/NoResults.js

```diff
@@ -13,15 +12,13 @@ const propTypes = {
 export default function NoResults(props) {
   const { label } = props;
   const [state] = useListBox();
-  const I18n = useI18n();
 
   if ((state.isOpen || state.isInline) && state.noResultsFound) {
     return (
       <>
         <sc.NoResults data-pka-anchor="no-results">{label}</sc.NoResults>
         <Toast kind={Toast.types.kind.VISUALLY_HIDDEN} aria-hidden>
-          {/* Using {label} here causes mysterious failures in specs */}
-          {I18n.t("listBox.filter.no_results_message")}
+          {label}
```

Comment:
is label passdown by I18n.t("listBox.filter.no_results_message")?

Reply:
`label={filter.props.noResultsMessage || I18n.t("listBox.filter.no_results_message")} />`

It's being passed like this